### PR TITLE
Bad dragon

### PR DESCRIPTION
**Pull request by Koriath**
Wouldn't be a chan server without these.

### CODE REVIEW

File: code/game/objects/items/baddragon.dm [hunk 1]
```diff
@@ -0,0 +1,125 @@
+//Buckle up, this is about to get really gay, really fast
+//Be glad it's not in the "toys.dm", this way you can just tick the file out
+
+//part of this was lifted from the baseball bat item. Seemed logical.
```
Comment (octareenroon91) [inline code/game/objects/items/baddragon.dm]: Wait do we even have baseball bats?

**Comment by octareenroon91** [comment]
See if you can get the mapmerge to work please.

**Comment by Koriath** [comment]
I'll give it a try.

**Comment by Koriath** [comment]
Should be good to go I think.

**Comment by octareenroon91** [comment]
If you made a change it's not pushed: Metastation dmm still shows:
 8,183 additions, 8,183 deletions not shown

**Comment by Koriath** [comment]
Yeah, forgot to click the damn sync button.

**Comment by octareenroon91** [comment]
That didn't seem to take.  Try this:
https://github.com/octareenroon91/-tg-station/raw/Kode/_maps/map_files/MetaStation/MetaStation.v40D.dmm

**Comment by Koriath** [comment]
Scratcg that, I gotta read the map merger documentation again.
The bats run and give okay messages, even find the right aditions to the map.

File: code/game/objects/items/baddragon.dm (hunk 1 above)
Comment (Koriath) [inline code/game/objects/items/baddragon.dm]: Not really, but I saw the code on the hippie station files.
Straight up lifted some lines from there as a template.
Maybe I'll port the baseball bats later. Would require some expansion of the throwing system they have I think.

**Comment by Koriath** [comment]
Wait, this should work. Map only says "142" additions now.

**Pull request merged**